docs/docker.md

```diff
@@ -85,8 +43,13 @@ so that you could easily locate it among your other containers.
 The `volumes-from` flag refers to Percona Server for MySQL and indicates
 that you intend to use the same data as the Percona Server for MySQL container.
 
-Run the container with exactly the same parameters that were used when the
-container was created:
+### Percona XtraBackup ARM64
+
+Starting with Percona XtraBackup 8.0.35-31, you can run the Docker ARM64 version of Percona XtraBackup. You can find the version and architecture on [Docker Hub Percona/Percona-Server Tags](https://registry.hub.docker.com/r/percona/percona-xtrabackup/tags?page=1&name=8.0). The Docker images for ARM64 version contain `aarch64` in the tag name, for example, `8.0.35-aarch64`.
```

Comment:
Should add the recommendation that we recommend using the same platform as the server source.

Reply:
Added